license-header

```diff
@@ -0,0 +1,2 @@
+Copyright (C) ${Year}, Ava Labs, Inc. All rights reserved.
+See the file LICENSE for licensing terms.
```

Comment:
nit: add new line at EOF (this can behave in annoying ways with some unix commands like diff for example) because the file spec technically requires a new line to denote EOF.